.github/workflows/package.yml

```diff
@@ -45,3 +45,28 @@ jobs:
           if [ "${IS_RC}" = "true" ]; then CONDA_LABEL="rc"; fi
           echo pushing ${{ github.ref }} with label $CONDA_LABEL
           anaconda upload --label $CONDA_LABEL conda.recipe/noarch/mypackagename*.tar.bz2
+
+  pypi-publish:
+    name: upload release to PyPI
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
Can we move this "if" to just the publish step? That way you always find out if the package builds

Reply:
Move the if check to be final step only.